Source/ULibs.TinyJsonDeser/JsonDeserializer.cs

```diff
@@ -144,7 +144,7 @@ public bool TryParseString(string json, out string output)
         /// <param name="json">The json string to parse.</param>
         /// <param name="output">If the parse succeeded, this will hold the parsed result.</param>
         /// <returns>Whether or not the parse attempt succeeded.</returns>
-        public bool TryParseArray(string json, out object[] output)
+        public bool TryParseArray(string json, out object?[]? output)

```

Comment:
Again, you're trying to state that the objects in the array will never be `null`, but that's simply not true. This library parses json, and `null` is a valid json type that can be returned. We could modify the library to return something like `JTokens` rather than built-in types, but that's a significant breaking change, and I'm not sure there's any call for it.